packages/unpublished-test-infra/addon-test-support/deprecated-test.js

```diff
@@ -33,10 +33,15 @@ export function deprecatedTest(testName, deprecation, testCallback) {
     }
   }
 
+  let testFn = test;
+  if (COMPAT_VERSION && gte(COMPAT_VERSION, VERSION)) {
```

Comment:
I don't fully grok this. How can `COMPAT_VERSION` ever be higher than the current version?

Reply:
COMPAT_VERSION can be set to any number. We use this to allow ourselves to aggressively strip everything by setting it to `99`.